assets/scss/base/_lists.scss

```diff
@@ -63,6 +94,23 @@ ul, nav ul {
 	}
 }
 
+/*
+Activity list
+
+An definiton list where the list uses the class `.activity-list` and the child elements uses the element class `.activity-list__date` and it's siblings use `.activity-list__description`
+
+Markup:
+<dl class="activity-list">
+  <dt class="activity-list__date">14th March 2016</dt>
+  <dd class="activity-list__description">description</dd>
+  <dd class="activity-list__description">description</dd>
+  <dd class="activity-list__description">description</dd>
+  <dd class="activity-list__description">description</dd>
+</dl>
+
+Styleguide Lists.activity list
+*/
+
 .activity-list {
 	padding-left: 0;
 	li {
```

Comment:
Also [`.activity-list__date--highlight`](https://github.com/hmrc/assets-frontend/pull/452/files#diff-49c36751aca4aa6941411171c9ce9318R129) hasn't been exampled. Neither has a [nested tag inside `.activity-list__date`](https://github.com/hmrc/assets-frontend/pull/452/files#diff-49c36751aca4aa6941411171c9ce9318R125) (even though they both do the same thing :unamused:)